ui/src/components/build/NoBuild.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import { Button } from 'patternfly-react';
+import StartBuildButton from '../../containers/StartBuildButton';
 
 import './NoBuild.css';
 
```

Comment:
We should probably convert non-state simple rendering components to the function-style. (as @josemigallas recommends).